javascript-conditionals/main.js

```diff
@@ -0,0 +1,102 @@
+/* exported isUnderFive,
+            isEven,
+            startsWithJ,
+            isOldEnoughToDrink,
+            isOldEnoughToDrive,
+            isOldEnoughToDrinkAndDrive,
+            categorizeAcidity,
+            introduceWarnerBro,
+            recommendMovie
+ */
+
+
+function isUnderFive(number){
+    if (number < 5 ) {
+        return true;
+    } else {
+        return false;
+    }
+};
+
+function isEven(number){
+    if ((number % 2) == 0){
+        return true;
+    } else {
+        return false;
+    }
+};
+
+function startsWithJ(string){
+    if (string[0] === 'J'){
+        return true;
+    } else {
+        return false;
+    }
+};
+
+function isOldEnoughToDrink(person){
+    if (person.age >= 21){
+        return true;
+    } else {
+        return false;
+    }
+};
+
+function isOldEnoughToDrive(person){
+    if (person.age >= 16){
+        return true;
+    } else {
+        return false;
+    }
+};
+
+function isOldEnoughToDrinkAndDrive(person){
+    if (person) {
+        return false;
+    }
+};
+
+function categorizeAcidity(pH){
+    if (pH === 7){
+        return 'neutral';
+    } else if (pH === 0 && pH <= 7) {
```

Comment:
This should be `>= 0` & `< 7`

Reply:
![javascript-conditionals](https://github.com/dennis-pnguyen/c0723-code-solutions/assets/122412863/7563c5ce-d0c2-4cf6-967d-85b5c2e1619b)

Updated, thanks!